validate.lic

```diff
@@ -551,6 +551,18 @@ class DRYamlValidator
     error('sell_loot_money_on_hand is invalid. The proper format is: <amount> <denomination> e.g. 3 silv or 4 bronze')
   end
 
+  def assert_that_pet_buffs_have_spell_requirements(settings)
+    return unless settings.buff_spells
+    
+    settings.buff_spells.select { |_name, data| data['pet_type'] }
+                        .reject{ |_name, data| data.include? ('recast_every') }
+                        .each{ |_name, data| error("Pet buffs require a custom recast timer. Please include a \"recast_every:\" setting for #{_name}.") }
```

Comment:
💡 By convention, the underscore in front of a variable name means it is not used. Since `_name` is used in the error message, inside the `.each` it should be named `name`. The same for the next block.

Reply:
Ah, this is something I couldn't find info for. I figured that the one without was the focus. Good to know this.